isos/bootstrap/bootstrap

```diff
@@ -51,11 +51,11 @@ if mount -t ext4 /dev/disk/by-label/containerfs ${MOUNTPOINT}; then
     #	rm -f /mnt/.tether/volumes/containerfs
 
     # set up the hostname
-    #	if [ -f /mnt/etc/hostname ]; then
-    #		/bin/hostname -F /mnt/etc/hostname
-    #	else
-    #		/bin/hostname -F /mnt/.tether-init/hostname
-    #	fi
+    if [ -f ${MOUNTPOINT}/etc/hostname ]; then
```

Comment:
Does the if on line 54 imply we may not have /etc/hostname file? If yes, would that be a problem?